--- bls-signatures-1.2.0/CMakeLists.txt.old	2024-04-05 07:25:56.663863766 +0100
+++ bls-signatures-1.2.0/CMakeLists.txt	2024-04-05 10:34:07.545680645 +0100
@@ -30,16 +30,16 @@
 
 include(FetchContent)
 
-FetchContent_Declare(Sodium
-  GIT_REPOSITORY https://github.com/AmineKhaldi/libsodium-cmake.git
-  # Latest commit at the moment this was added here
-  # Anchored to libsodium v1.0.18
-  GIT_TAG f73a3fe1afdc4e37ac5fe0ddd401bf521f6bba65
-)
+#FetchContent_Declare(Sodium
+#  GIT_REPOSITORY https://140.82.121.4/robinlinden/libsodium-cmake.git
+#  # Latest commit at the moment this was added here
+#  # Anchored to libsodium v1.0.18
+#  GIT_TAG f73a3fe1afdc4e37ac5fe0ddd401bf521f6bba65
+#)
 set(SODIUM_PCH "on" CACHE STRING "")
 set(SODIUM_DISABLE_TESTS "on" CACHE STRING "")
 set(SODIUM_CHIA_MINIMAL "on" CACHE STRING "")
-FetchContent_MakeAvailable(Sodium)
+#FetchContent_MakeAvailable(Sodium)
 
 if (DEFINED ENV{RELIC_MAIN})
   set(RELIC_GIT_TAG "origin/main")
